modules/core/src/test/java/org/apache/ignite/testframework/junits/GridAbstractTest.java

```diff
@@ -437,6 +446,18 @@ protected IgniteLogger log() {
         return log;
     }
 
+    /**
+     * Sets the log level for root logger ({@link #log}) to {@link Level#DEBUG}. The log level will be resetted to
+     * default in {@link #afterTest()}.
+     */
+    protected final void setRootLoggerDebugLevel() {
+        Logger logger = Logger.getRootLogger();
+
+        assertNull(logger + " level: " + Level.DEBUG, changedLevels.put(logger, logger.getLevel()));
```

Comment:
I think it is not necessary to throw an error if you set debag twice.

Reply:
You make a mistake if you set a debug level twice in one test. {{changedLevels}} cleans after each test (see afterTest())

Reply:
At the discretion of core reviewer.